ortools/linear_solver/xpress_interface.cc

```diff
@@ -2246,6 +2248,8 @@ bool XpressMPCallbackContext::CanQueryVariableValues() {
   return Event() == MPCallbackEvent::kMipSolution;
 }
 
+double XpressInterface::infinity() { return std::max(XPRS_PLUSINFINITY, -XPRS_MINUSINFINITY); }
```

Comment:
If `XPRS_PLUSINFINITY != -XPRS_MINUSINFINITY`, it means that we are more or less restrictive than XPRES on what is infinite and what is not.

I checked in xprs.h that as of v9.2.5, both values have the same absolute value so it won't matter for now.

Reply:
same as above, we need to be sure that solver_infinity is larger than XPRS_PLUSINFINITY **AND** -XPRS_MINUSINFINITY

Reply:
OK !